unittest/archive.etest

```diff
@@ -1501,84 +1501,3 @@ ETEST_archive_xdiff()
         done
     done
 }
-
```

Comment:
This test was added years and years ago when bootable ISOs were a new thing and we wanted a test to verify it. But this is a huge integration test not a unit test. And it requires virtualbox, which is gross. I'm removing it.